Fix installation prefix and build with installation rpath, so that rpath is
correct. Running rospack from the build dir is handled via a {DY,}LD_LIBRARY_PATH

--- tools/rospack/CMakeLists.txt~       2011-11-17 22:16:21.000000000 +0100
+++ tools/rospack/CMakeLists.txt        2011-11-17 22:18:09.000000000 +0100
@@ -9,7 +9,8 @@
 
 set(CMAKE_OSX_ARCHITECTURES "x86_64")
 
-set(CMAKE_INSTALL_PREFIX /tmp/rospack)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
+set(CMAKE_INSTALL_PREFIX $ENV{PREFIX}/ros/tools/rospack)
 #set(CMAKE_INSTALL_RPATH_USE_LINK_RPATH true)
 #set(CMAKE_SKIP_BUILD_RPATH true)
 set(CMAKE_INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}/lib")
